thunder/core/jit_ext.py

```diff
@@ -617,6 +617,10 @@ def _general_jit_torch_autograd_function_apply_lookaside(obj: Any, *args, **kwar
     ctx_proxy = proxy(ctx, name=None, history=None)
     wrapped_ctx = wrap_const(ctx_proxy)
     custom_forward_result = _interpret_call(custom_forward, wrapped_ctx, *args, **kwargs)
+    check(
+        custom_forward_result != INTERPRETER_SIGNALS.EXCEPTION_RAISED,
+        lambda: f"{custom_autograd_function_cls}.forward could not be traced by Thunder.",
+    )
```

Comment:
or to return stack traces for better information?
```suggestion
    if custom_forward_result == INTERPRETER_SIGNALS.EXCEPTION_RAISED:
        return custom_forward_result
```

Reply:
I have tended to use `is`, but yes, this is how to propagate the  exception.

Reply:
oops, thanks